src/applications/find-forms/helpers/index.js

```diff
@@ -75,10 +75,8 @@ export const sortTheResults = (sortByPropertyName, indexA, indexB) => {
  * @param {string|number} index index where we need to add the dash
  * @return {string} new substring with dash
  */
-export const regexpDashAdder = (match, index) => {
-  const regex = new RegExp(`(?<=^.{${index}})`);
-  return match.replace(regex, '-');
-};
+export const regexpDashAdder = (match, index) =>
+  `${match.slice(0, index)}-${match.slice(index)}`;
 
```

Comment:
If someone knows of a good regexp, I am up for trying it again. I couldn't figure another out.

Reply:
How you have it now is exactly how I would do it! Regex can be very cryptic and sometimes very costly performance-wise. Simple string arithmetic like this is 💯

Reply:
you could use `[].join` if you don't like the way the template string looks -
```js
[
    match.slice(0, index),
    match.slice(index)
].join('-);
```